packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -65,14 +67,14 @@ const Encryption = WebexPlugin.extend({
         .then((result) => result.plaintext.toString()));
   },
 
-  download(scr) {
+  download(scr, optional) {
```

Comment:
As @andrewholsted mentioned, JSDoc comments would help contributors understand what this method is doing, and what it requires.